nvflare/private/fed/client/communicator.py

```diff
@@ -195,25 +195,26 @@ def pull_task(self, servers, project_name, token, ssid, fl_ctx: FLContext):
             size = len(task.payload)
             task.payload = fobs.loads(task.payload)
             task_name = task.payload.get_header(ServerCommandKey.TASK_NAME)
+            task.payload.add_cookie(ReservedHeaderKey.SSID, ssid)
```

Comment:
Change to use fl_ctx, not cookie, which is a kludge.

Reply:
thanks, fixed using fl_ctx.